lib/node_modules/@stdlib/math/base/special/cabsf/package.json

```
@@ -66,5 +66,147 @@
     "complex",
     "cmplx",
     "number"
-  ]
+  ],
+  "__stdlib__": {
+    "scaffold": {
+      "$schema": "math/base@v1.0",
+      "base_alias": "cabs",
```

```suggestion
      "base_alias": "abs",
```
I believe we should have the base alias as `abs` here. For reference, see that `cabsf` is listed as a scalar kernel of `abs` [here](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/special/data/unary_function_database.json#L19).